src/hotspot/os/windows/os_windows.cpp

```diff
@@ -866,8 +866,8 @@ bool os::win32::available_memory(size_t& value) {
     value = static_cast<size_t>(ms.ullAvailPhys);
     return true;
   } else {
-    assert(false, "GlobalMemoryStatusEx failed in win32::available_memory().");
-    return res == TRUE;
+    assert(false, "GlobalMemoryStatusEx failed in win32::available_memory(): %u", ::GetLastError());
```

Comment:
```suggestion
    assert(false, "GlobalMemoryStatusEx failed in win32::available_memory(): %lu", ::GetLastError());
```
There is no consistency here but the primary ways we print `GetLastError` are using `%lu`, `%ld` or `%d`.

Reply:
Addressed by usage of `%lu`.